resources/admin.py

```diff
@@ -35,17 +38,71 @@ class BoatTypeAdmin(TranslatableAdmin):
     pass
 
 
-class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin):
-    list_display = ("name",)
+class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin, HarborManager):
```

Comment:
why do you need the `HarborManager` here? Django will do `Harbor.objects.all()` or `Harbor.objects.get(pk=obj.id)`, which will already have all the annotations. 🤔

Reply:
It was part of my experiments trying to get the new annotated fields to the admin site, gone now. 👍